source/github-helpers/add-button-next-to-branch-select-menu.ts

```diff
@@ -0,0 +1,16 @@
+import select from 'select-dom';
+import elementReady from 'element-ready';
+
+export default async function addButtonNextToBranchSelectMenu(button: Element): Promise<void> {
```

Comment:
I don't think we need a new file. Export it from one of the other feature files.